scripts/example_relationships_establishments.py

```diff
@@ -1,8 +1,79 @@
 import json
+import re
+import pdb
 
+def extract_function_name(line):
```

Comment:
Maybe move this to the other script as this script is supposed to establish relationships and not do something like this??

Reply:
Left it be for nw.